src/unlock_region.c

```diff
@@ -298,6 +299,32 @@ EFI_STATUS print_legacy_region_info(EFI_LEGACY_REGION2_PROTOCOL *legacy_region)
     return EFI_SUCCESS;
 }
 
+static bool test_bios_region_rw(void) {
+    uint32_t *bios_region = (uint32_t *)BIOSROM_START;
+    uint32_t *bios_region_end = (uint32_t *)BIOSROM_END;
+    uint32_t *ptr = bios_region;
+
+    while (ptr < bios_region_end) {
+        uint32_t val;
```

Comment:
Using an unutilised stack variable here is a little bit wild.
Better to use a known initial pattern, maybe.

Reply:
I fixed this as you saw fit :)